src/store/models/lightning.ts

```diff
@@ -89,7 +89,7 @@ export interface LightningModel {
 const lightningModel: LightningModel = {
   // state properties
   nodes: {},
-  // reducer actions (mutations allowed thx to immer)
+
```

Comment:
Did you intentionally remove this comment?

Reply:
No. that was a mistake. Let me add it back